.github/workflows/package.yml

```diff
@@ -64,9 +74,30 @@ jobs:
       run: ${{matrix.CMD_BUILD}}
 
     - name: Validate package for ${{matrix.TARGET}}
+      if: matrix.TARGET != 'macos'
       run: ./dist/${{ matrix.TARGET }}/${{matrix.OUT_FILE_NAME}}
 
-    - uses: actions/upload-artifact@v4
+    - name: Validate package for Mac
+      if: matrix.TARGET == 'macos'
+      run: ./dist/${{ matrix.TARGET }}/${{ matrix.OUT_FILE_NAME }}/Contents/MacOS/${{ matrix.APP_BINARY_FILE_NAME }}
```

Comment:
This seems to imply that the package won't run as uploaded. What does a user have to do with the file on our release page in order to run it?

Reply:
My understanding is as below:

-  After downloading the artifact from the release page, unzip the downloaded file, move the tabcmd.app file to Mac's Applications folder
-  Run the application by double clicking on the app icon on the Applications folder, in this case given that it is a CLI based app, users would invoke the binary from /Applications/tabcmd.app/Contents/MacOS/tabcmd

Does that sound as the right user flow, if not please let me know, I'm happy to change accordingly

Reply:
That sounds like a reasonable flow for Mac. We just have to make sure that we explain this in the README and maybe on the release page.